WordPressKitTests/ReaderPostServiceRemote+CardsTests.swift

```diff
@@ -48,7 +48,7 @@ class ReaderPostServiceRemoteCardTests: RemoteTestCase, RESTTestable {
 
         readerPostServiceRemote.fetchCards(for: ["cats"], success: { cards, _ in
             let postTypes = cards.map { $0.type }
-            let expectedPostTypes: [RemoteReaderCard.CardType] = [.interests, .unknown, .post, .post, .post, .post, .post, .post, .post, .post]
+            let expectedPostTypes: [RemoteReaderCard.CardType] = [.interests, .sites, .post, .post, .post, .post, .post, .post, .post, .post]
```

Comment:
Maybe we can add a test that checks for the content of the `.sites` object? Up to you though, and I don't think we need to test all the properties.

Reply:
I opted against doing this since testing the content of the sites object should be handled in its own test in `RemoteReaderSiteTopic` vs in the cards service.